scripts/sct_register_to_template.py

```diff
@@ -241,7 +253,10 @@ def main(args=None):
     zsubsample = param.zsubsample
 
     # retrieve template file names
-    file_template_vertebral_labeling = get_file_label(os.path.join(path_template, 'template'), 'vertebral labeling')
+    if label_type == 'spinal':
+        file_template_labeling = get_file_label(os.path.join(path_template, 'template'), 'spinal body labeling')
```

Comment:
ah! my refactoring of the field "name" in the file `info_label.txt` will break this part of the code. Instead of fetching file name by field "name", we should use the field "ID", so we have more flexibility to alter the file `info_label.txt`. What do you think @charleygros? That would imply a minor refactoring of the function `get_file_label()` to input ID instead of name (then we need to check where the function is used).

Reply:
Seems a good idea to me!
Was wondering why we prefered the names instead of the IDs. It was maybe for this case: [here](https://github.com/neuropoly/spinalcordtoolbox/blob/4c2aca38d2536e202d58b9be646652a0f4280b6a/spinalcordtoolbox/vertebrae/core.py#L67). Was it?

Reply:
Happy to work on this!

Reply:
After more thoughts, I think that fetching by name was easier than ID (not self explanatory).
Alternatively, we could go with the IDs, and add a comment on those modified lines, something like:
```python
          fname_src,
          spinalcordtoolbox.metadata.get_file_label(path_template, '1', output="filewithpath"),  # label = 'T2-weighted template'
          spinalcordtoolbox.metadata.get_file_label(path_template, '5', output="filewithpath"),  # label = 'gray matter mask (probabilistic)'
          spinalcordtoolbox.metadata.get_file_label(path_template, '4', output="filewithpath")  # label = 'white matter mask (probabilistic)'
```
What do you think @jcohenadad ?

Reply:
Yes, commenting the code is exactly what I had in mind 👍

Reply:
Super!
Done